### ISSUE TITLE
Steam 1.16.10 - VIVE Pro - segmentation fault in LfMutexUnlockRobust (example binary)

### ISSUE THREAD

**Issue by jeffreyfq** [ValveSoftware/openvr#1564]
The examples hellovr_opengl and the Vulkan version crash on steam 1.16.10, Ubuntu 20.04, 'linux64'

(before updating steamvr, it was working on my machine, I confirmed this on two different machines, but Ubuntu 20.04.
I am not sure what my previous version of steamvr was as it performed an auto-update on start-up ... )

The complete console output is here: [log.txt](https://github.com/ValveSoftware/openvr/files/6781743/log.txt)

lldb debug output:
```
Process 519762 stopped
* thread #4, name = 'hellovr_opengl', stop reason = signal SIGSEGV: invalid address (fault address: 0x7f691c22f9d8)
    frame #0: 0x00007ffff3786ad8 vrclient.so`LfMutexUnlockRobust(LfMutex*) + 56
vrclient.so`LfMutexUnlockRobust:
->  0x7ffff3786ad8 <+56>: movq   %rax, -0x8(%rdx)
    0x7ffff3786adc <+60>: movq   0x20(%r12), %rdx
    0x7ffff3786ae1 <+65>: andq   $-0x2, %rax
    0x7ffff3786ae5 <+69>: movq   %rdx, (%rax)
(lldb) bt
* thread #4, name = 'hellovr_opengl', stop reason = signal SIGSEGV: invalid address (fault address: 0x7f691c22f9d8)
  * frame #0: 0x00007ffff3786ad8 vrclient.so`LfMutexUnlockRobust(LfMutex*) + 56
    frame #1: 0x00007ffff36f990a vrclient.so`CSyncLockThread::Run() + 154
    frame #2: 0x00007ffff37653d9 vrclient.so`SteamThreadTools::CThread::ThreadProc(void*) + 201
    frame #3: 0x00007ffff74a1609 libpthread.so.0`start_thread(arg=<unavailable>) at pthread_create.c:477:8
    frame #4: 0x00007ffff76cb293 libc.so.6`__clone + 67
```

This seems to be a similar but old issue, but maybe it is related ?!:
[https://forum.vive.com/topic/9426-programs-that-use-the-openvr-sdk-to-display-some-image-on-the-vive-pro-may-terminated-abnormally/](https://forum.vive.com/topic/9426-programs-that-use-the-openvr-sdk-to-display-some-image-on-the-vive-pro-may-terminated-abnormally/)

**Comment by kisak-valve** [comment]
Possible duplicate of #1533.

**Comment by jeffreyfq** [comment]
It turns out that steamvr version 1.16.10 was most likely the previous version and the installation was not completed, recently steam asked for root permissions and to finalize the installation process. Now it is shown that version 1.18.6 is installed.

The openvr samples and my own code do not cause errors anymore, so it seems that an incomplete update caused the trouble. ( closed #1564 )
